svelte-kit-scss/src/routes/(authenticated)/[username]/[repo]/[issueSearchType=issue_search_type]/+page.svelte

```diff
@@ -23,7 +26,11 @@
       {milestoneFilters}
       {labelFilters}
     />
-    <IssueSearchList items={issues} />
+    {#if hasIssues}
+      <IssueSearchList items={issues} />
+    {:else}
+      <ListBlankSlate heading="No results matched your search." icon={IssueOpened24} />
+    {/if}
     <div slot="bottom" class="bottom">
```

Comment:
Shall we move the pagination to inside the else-block as well? There are no pages to go through when we have no results.